Formula/vkectl.rb

```diff
@@ -0,0 +1,32 @@
+class Vkectl < Formula
+  desc "Command-Line Interface for VKE(VolcanoEngine Kubernetes Engine)"
+  homepage "https://github.com/volcengine/vkectl"
+  url "https://github.com/volcengine/vkectl/archive/refs/tags/v0.1.1.tar.gz"
+  sha256 "15f0f3786c03d53702306ba4ae8812afe59e0094356d1202c292cca87242ac77"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w -X github.com/volcengine/vkectl/pkg/version.version=v#{version}"), "./main"
+  end
+
+  test do
+    version_out = shell_output("#{bin}/vkectl version")
+    assert_match version.to_s, version_out
+
+    resource_help_out = shell_output("#{bin}/vkectl resource -h")
+    assert_match "AddNodes", resource_help_out
+    assert_match "AddVciSubnets", resource_help_out
+    assert_match "ListClusters", resource_help_out
+    assert_match "CheckCidrConflict", resource_help_out
+    assert_match "CreateNodePool", resource_help_out
+
+    resource_get_addon_out = shell_output("#{bin}/vkectl resource GetAddon")
+    assert_match '{"Name":"","Version":"","Status":"","ErrorMessage":""}', resource_get_addon_out
+
+    security_get_check_item_out = shell_output("#{bin}/vkectl security GetCheckItem")
+    assert_match '{"Number":"","Description":"","Rationale":"","Remediation":"","Impact":""}',
+      security_get_check_item_out
```

Comment:
Is this supposed to be JSON? If so, we can just do something like
```suggestion
    resource_get_addon_out = JSON.parse(shell_output("#{bin}/vkectl resource GetAddon"))
    assert_empty resource_get_addon_out["Name"]

    security_get_check_item_out = JSON.parse(shell_output("#{bin}/vkectl security GetCheckItem"))
    assert_empty security_get_check_item_out["Number"]
```
That way we just check it's valid JSON and check one field each for what we expect here.

Reply:
yeah, it will indeed only be JSON, thanks for the suggestion